src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.onclick = function (e) {
```

Comment:
Ensure that the 'thumbs' element exists before attaching the onclick event handler. If 'thumbs' is null, this will throw an error. Consider adding a null check for 'thumbs' before setting the onclick function.